main.js

```diff
@@ -1,6 +1,31 @@
 require('es6-promise').polyfill();
 var rest = require('restler');
 
+// Object.assign polyfill
+if (typeof Object.assign != 'function') {
```

Comment:
How about using the object-assign ponyfill npm package instead? It has a ton of little fixes for faulty native `Object.assign`.